server/events.py

```diff
@@ -115,8 +115,11 @@ class Event(BaseModel):
     status_code: StatusCode
 
     @validator("data", pre=True)
-    def valid_data(cls, value, values):
+    def valid_data(cls, value: EventData | Mapping, values):
```

Comment:
`Type[EventData] | Mapping`

Reply:
We won't receive the object's type, but the object itself though.

Reply:
Nvm, don't do this :p